go.mod

```diff
@@ -1,35 +1,36 @@
 module github.com/envoyproxy/ai-gateway
 
-go 1.25
+// Explicitly specify the Go patch version to be able to purge the CI cache correctly.
+go 1.25.1
```

Comment:
Note: not sure why but apparently the CI didn't automatically use 1.25 hence Trivy reported some CVE (which is not applicable to this project though) on main branch.